inlong-agent/agent-common/src/main/java/org/apache/inlong/agent/constant/AgentConstants.java

```diff
@@ -171,12 +171,26 @@ public class AgentConstants {
     public static final String DEFAULT_COMPRESSION_TYPE = "NONE";
 
     public static final String PULSAR_CLIENT_PRODUCER_NUM = "agent.sink.pulsar.producer.num";
+    public static final String KAFKA_SINK_PRODUCER_NUM = "agent.sink.kafka.producer.num";
```

Comment:
Suggested naming it like Pulsar, for example:

```suggestion
    public static final String KAFKA_CLIENT_PRODUCER_NUM = "agent.sink.kafka.producer.num";
```

Reply:
The usage of pulsar and kafka is different. To send data using pulsar, need to first create a client; Sending data using Kafka does not require creating a client, just creating a producer. In Kafka, not using the `CLIENT` variable name may be more suitable.

Reply:
Okay.